.github/workflows/lint-validate.yml

```
@@ -35,19 +35,19 @@ jobs:
           yamllint -d scripts/yamllint-config.yaml --no-warnings config/
 
       - name: Unlock git-crypt secrets
-        if: github.event.pull_request.head.repo.fork == 'false'
+        if: github.event.pull_request.head.repo.fork == false
```

What about explicit boolean not with `!`?

```suggestion
        if: !github.event.pull_request.head.repo.fork
```

---

I tried it, but it turns out not to be valid in the GHA schema

---

Okay, then! The docs suggest it should be, so maybe a bug in the schema.